app/src/test/java/ch/epfl/sdp/drone3d/model/mission/ParallelogramMissionBuilderTest.kt

```diff
@@ -9,18 +9,27 @@ import ch.epfl.sdp.drone3d.model.mission.ParallelogramMissionBuilder.Companion.b
 import ch.epfl.sdp.drone3d.model.mission.ParallelogramMissionBuilder.Companion.buildSinglePassMappingMission
 import org.junit.Assert.assertEquals
 import org.junit.Test
+import kotlin.math.PI
 
 class ParallelogramMissionBuilderTest {
+
+    companion object{
+        private const val cameraAngle = (PI/2).toFloat()
```

Comment:
Would be nice to be coherent and name it cameraPitch

Reply:
This is done in another PR but I will do it here for consistency